frontend/src/Editor/Container.jsx

```diff
@@ -563,7 +567,7 @@ export const Container = ({
     removeComponent,
     currentLayout,
     deviceWindowWidth,
-    selectedComponents,
+    useEditorStore.getState().selectedComponents,
```

Comment:
Here as well

Reply:
selectedComponents not being used from containerProps, hence removed it from containerProps.